pkg/api/common.go

```diff
@@ -191,8 +193,14 @@ func respond(w http.ResponseWriter, status int, message interface{}) {
 	w.Header().Set("Content-Type", "application/json")
 	w.Header().Set("Cache-Control", "no-store")
 	w.WriteHeader(status)
+
+	statusText := "success"
+	if status != http.StatusOK {
+		statusText = http.StatusText(status)
+	}
```

Comment:
Why this is needed? Is `ok` returned by `http.StatusText` not ok?

Reply:
No, because the ideal response in `status` field should be `success` to match with Prometheus output.

Reply:
Could you add a comment in code with that explanation?

Reply:
Sure